Azure-Samples/graphrag-accelerator
# removed all webhook references

# How to test
Deploy a brand new cluster and look at the APIM `/index` API definition once the APIM instance is running. You should not see any parameter references to `webhook_url`.

To see the APIM API doc page, you can visit `https://<apim_endpoint>/manmgape/docs`.

## Notes
- Make sure you are running the Azure VPN before testing the API.
